tests/test_modeling_xlnet.py

```diff
@@ -593,6 +593,14 @@ def test_retain_grad_hidden_states_attentions(self):
         # xlnet cannot keep gradients in attentions or hidden states
         return
 
+    def _check_attentions_for_generate(self, *args, **kwargs):
+        # TODO(PVP)
+        pass
+
+    def _check_hidden_states_for_generate(self, *args, **kwargs):
```

Comment:
xlnet has dual-stream architecture which leads to special attentions and hidden_states